### DISCUSSION TITLE
Alaska not reporting?

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [weather-gov/api#349] [Q&A]
Hi, I'm trying to use the API to gather forecast information for Eagle River, AK (Anchorage) in Home Assistant, but am getting an error. When I put in the following URL into my browser directly

https://api.weather.gov/points/61.3142,-149.4848

everything from **gridx:** to **observationStations:** is "null". If is use the example lat/long for Washington, I get valid URLs that then provide the forecast data. Are you reporting forecast data for Alaska? Thanks. Shane

Here's the output from the above URL:

{
  "@context": [
    "https://raw.githubusercontent.com/geojson/geojson-ld/master/contexts/geojson-base.jsonld",
    {
      "wx": "https://api.weather.gov/ontology#",
      "s": "https://schema.org/",
      "geo": "http://www.opengis.net/ont/geosparql#",
      "unit": "http://codes.wmo.int/common/unit/",
      "@vocab": "https://api.weather.gov/ontology#",
      "geometry": {
        "@id": "s:GeoCoordinates",
        "@type": "geo:wktLiteral"
      },
      "city": "s:addressLocality",
      "state": "s:addressRegion",
      "distance": {
        "@id": "s:Distance",
        "@type": "s:QuantitativeValue"
      },
      "bearing": {
        "@type": "s:QuantitativeValue"
      },
      "value": {
        "@id": "s:value"
      },
      "unitCode": {
        "@id": "s:unitCode",
        "@type": "@id"
      },
      "forecastOffice": {
        "@type": "@id"
      },
      "forecastGridData": {
        "@type": "@id"
      },
      "publicZone": {
        "@type": "@id"
      },
      "county": {
        "@type": "@id"
      }
    }
  ],
  "id": "https://api.weather.gov/points/61.3142,-149.4848",
  "type": "Feature",
  "geometry": {
    "type": "Point",
    "coordinates": [
      -149.4848,
      61.3142
    ]
  },
  "properties": {
    "@id": "https://api.weather.gov/points/61.3142,-149.4848",
    "@type": "wx:Point",
    "cwa": "AER",
    "forecastOffice": "https://api.weather.gov/offices/AFC",
    "gridX": null,
    "gridY": null,
    "forecast": null,
    "forecastHourly": null,
    "forecastGridData": null,
    "observationStations": null,
    "relativeLocation": {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          -149.284329,
          61.17425
        ]
      },
      "properties": {
        "city": "Anchorage",
        "state": "AK",
        "distance": {
          "value": 18898.91434010521,
          "unitCode": "unit:m"
        },
        "bearing": {
          "value": 325,
          "unitCode": "unit:degrees_true"
        }
      }
    },
    "forecastZone": "https://api.weather.gov/zones/forecast/AKZ101",
    "county": "https://api.weather.gov/zones/county/AKC020",
    "fireWeatherZone": "https://api.weather.gov/zones/fire/AKZ101",
    "timeZone": "America/Anchorage",
    "radarStation": "PAHG"
  }
}

**Comment by StephenClouse** [comment]
The raw data is there, but there's a bug with the resolution from lat/lon to gridpoint for Alaska. We're working on getting a fix out. Unfortunately we don't have a workaround for this issue.

**Comment by GrizzlyAK** [comment]
Ok, thanks Stephen. I wasn’t sure if it was something I was doing wrong or a problem on your end. If you need someone to help test, debug, or review code, I’d be happy to help. I currently have plenty of time! 

Cheers,
Shane

**Comment by dejab** [comment]
Can I assume the same bug you mention above is related to the issue with:

https://api.weather.gov/points/39.3875,-119.649

Resulting in:

An error occurred.
Sorry, the page you are looking for is currently unavailable.
Please try again later.

If you are the system administrator of this resource then you should check the error log for details.

Faithfully yours, nginx.

**Comment by StephenClouse** [comment]
No, that's an outage and should be reported to the NCO infrastructure team as described here https://weather-gov.github.io/api/reporting-issues

**Comment by GrizzlyAK** [comment]
Do you know if the issue of reporting in AK has been resolved? Didn’t know if closure of the ticket meant so, or if it was a timeout thing. Thanks.
